src/main/resources/db/changelog/init/sql/insert_data.sql

```diff
@@ -4,37 +4,43 @@ VALUES (1, 'USER', current_timestamp, current_timestamp);
 INSERT INTO role (rol_id, rol_name, rol_created, rol_modified)
 VALUES (2, 'ADMIN', current_timestamp, current_timestamp);
 
-INSERT INTO location (loc_id, loc_country, loc_city, loc_created, loc_modified)
-VALUES (1, 'Ukraine', 'Kyiv', current_timestamp, current_timestamp);
+INSERT INTO country (country_code, country_full_name)
+VALUES ('UA', 'Ukraine');
 
-INSERT INTO location (loc_id, loc_country, loc_city, loc_created, loc_modified)
-VALUES (2, 'Ukraine', 'Lviv', current_timestamp, current_timestamp);
+INSERT INTO country (country_code, country_full_name)
+VALUES ('BY', 'Belarus');
 
-INSERT INTO location (loc_id, loc_country, loc_city, loc_created, loc_modified)
-VALUES (3, 'Ukraine', 'Kharkiv', current_timestamp, current_timestamp);
+INSERT INTO location (loc_id, country_code, loc_city, loc_address_line, loc_created, loc_modified)
+VALUES (1, 'UA', 'Kyiv', 'Khreshchatyk, 38', current_timestamp, current_timestamp);
 
-INSERT INTO location (loc_id, loc_country, loc_city, loc_created, loc_modified)
-VALUES (4, 'Ukraine', 'Rivne', current_timestamp, current_timestamp);
+INSERT INTO location (loc_id, country_code, loc_city, loc_address_line, loc_created, loc_modified)
+VALUES (2, 'UA', 'Lviv', 'Sichovyh Strilciv, 54', current_timestamp, current_timestamp);
 
-INSERT INTO location (loc_id, loc_country, loc_city, loc_created, loc_modified)
-VALUES (5, 'Ukraine', 'Chernigov', current_timestamp, current_timestamp);
+INSERT INTO location (loc_id, country_code, loc_city, loc_address_line, loc_created, loc_modified)
+VALUES (3, 'BY', 'Minsk', 'Independence Avenue, 1', current_timestamp, current_timestamp);
 
-INSERT INTO location (loc_id, loc_country, loc_city, loc_created, loc_modified)
-VALUES (6, 'Belarus', 'Minsk', current_timestamp, current_timestamp);
+INSERT INTO location (loc_id, country_code, loc_city,loc_address_line, loc_created, loc_modified)
+VALUES (4, 'Ukraine', 'Rivne', 'Beregova, 13', current_timestamp, current_timestamp);
```

Comment:
countryCode should be used as a foreignKey, not countryFullName
same below

Reply:
Changed